src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -66,7 +71,7 @@ public HubConnection(IConnectionFactory connectionFactory, IHubProtocol protocol
             await StartAsyncCore(cancellationToken).ForceAsync();
         }
 
-        public async Task StopAsync()
+        public async Task StopAsync(CancellationToken cancellationToken = default)
```

Comment:
Added cancellationToken but it doesn't actually do anything right now. What should it do?

Reply:
Not sure yet. Need to look deeper. Maybe we need to pass it to DisposeAsync or use it from the outside to cancel dispose. But we can just not support it for now. File an issue.

Reply:
https://github.com/aspnet/SignalR/issues/2017